README.md

```diff
@@ -1,68 +1,48 @@
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+#### Recommended extensions for VSCodium
 
-Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+To install these extensions please see [Visual Studio Code's documentation](https://code.visualstudio.com/docs/introvideos/extend). If you prefer not to visit sites with proprietary software, do the following steps:
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+1. Open the command palette and choose `Extensions: Install extensions`.
+2. Search for your extension.
+3. Click / select it.
+4. Press install.
 
-## Learn More
+##### Extensions for linting purposes
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+- [ESLint](https://open-vsx.org/extension/dbaeumer/vscode-eslint)
+- [Prettier - Code formatter](https://open-vsx.org/extension/esbenp/prettier-vscode)
+- [sort-imports](https://github.com/amatiasq/vsc-sort-imports)
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+Sidenote: Many required extensions are already built-in in VSCode / VSCodium, such as: [TypeScript Language Basics](https://open-vsx.org/extension/vscode/typescript). The necessity for these extensions are not documented.
 
-### Code Splitting
+### License 📝
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/code-splitting
-
-### Analyzing the Bundle Size
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size
-
-### Making a Progressive Web App
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app
-
-### Advanced Configuration
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration
-
-### Deployment
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
-
-### `yarn build` fails to minify
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
+This repository is [GNU AFFERO GENERAL PUBLIC LICENSE](./LICENSE).
```

Comment:
I think perhaps you want to add the license to the project too! :)

Reply:
What liscens should we have? MIT or AGPL?